lib/Serge/Engine/Plugin/parse_csv.pm

```diff
@@ -0,0 +1,273 @@
+package Serge::Engine::Plugin::parse_csv;
+
+use 5.10.0;
+use strict;
+use warnings;
+
+use parent 'Serge::Engine::Plugin::Base::Parser';
+
+use English qw( -no_match_vars );
+use Data::Dumper;
+use IO::String;
+use Serge::Mail;
+use Serge::Util qw(xml_escape_strref);
+use Text::CSV_XS;
+
+sub name {
+    return 'CSV Parser Plugin';
+}
+
+sub init {
+    my $self = shift;
+
+    $self->SUPER::init(@_);
+
+    $self->merge_schema({
+        email_from    => 'STRING',
+        email_to      => 'ARRAY',
+        email_subject => 'STRING',
+
+        end_of_line    => 'STRING',
+        delimiter      => 'STRING',
+        escape         => 'STRING',
+        quote          => 'STRING',
+
+        column_key     => 'STRING',
```

Comment:
I think it makes sense in the context of this PR, but if it's easy for you to do in a follow-up PR, I'm fine with that.